guides/bug_report_templates/integration_test_master.rb

```diff
@@ -0,0 +1,109 @@
+begin
+  require 'bundler/inline'
+rescue LoadError => e
+  $stderr.puts 'Bundler version 1.10 or later is required. Please update your Bundler'
+  raise e
+end
+
+gemfile(true) do
+  source 'https://rubygems.org'
+  # Activate the gem you are reporting the issue against.
+  gem 'rails', '4.2.0'
+  gem 'devise'
+  gem 'sqlite3'
+  gem 'byebug'
+end
+
+require 'rack/test'
+require 'action_controller/railtie'
+require 'active_record'
+require 'devise/rails/routes'
+require 'devise/rails/warden_compat'
+
+ActiveRecord::Base.establish_connection( adapter: :sqlite3, database:  "test.sqlite")
```

Comment:
can we use the in memory database instead, like the AR templates does? This way we don't to worry about the table already existing like on line 28.